src/index.js

```diff
@@ -572,6 +612,27 @@ const indy = {
     )
   },
 
+  async appendTxnAuthorAgreementAcceptanceToRequest(request: LedgerRequest, text: string, version: string, taaDigest: string, mechanism: string, time: number): Promise<string> {
```

Comment:
I think the string return type is not correct here

Reply:
Updated